### DISCUSSION TITLE
아카라이브 각 채널 자체 광고들은 허용하도록 만들려면 어떤 스크립트를 설정해야할까요?

### DISCUSSION THREAD

**Discussion by Flaymore1211** [List-KR/community#57] [Q&A]
아카라이브 채널광고들은 크게 사이트 이용에 지장이 가는 수준도 아니고 아카라이브 각 채널들의 다채롭고 재밌는 광고들도 매력의 요소라 생각해서 해당 부분만 광고차단에서 예외로 설정하고 싶습니다.
사진으로 첨부했습니다. 아카라이브 채널마다 빨간색 표시된 부분만큼은 표시되도록 허용하고 싶습니다.
![새 프로젝트](https://github.com/List-KR/community/assets/124432684/b7486359-cf9f-430e-bc5d-299fb57c8914)

**Comment by piquark6046** [comment]
테스트해보세요:
```adblock
arca.live#@#div.ad
||arca.live/api/*pagead?$xmlhttprequest,badfilter
||da.namu.la^$domain=arca.live,badfilter
```

**Reply by Flaymore1211** [reply]
감사합니다. 덕분에 해결되었습니다.
적용하니 사진에 있는 것과 같은 좌우의 커다란 광고들이 다시 나타나긴 했지만 애드가드 브라우저 어시스턴트 확장프로그램을 통해 재차단을 했습니다. 재차단을 했을시 다음과 같은 스크립트들을 추가하면 되겠더라구요

```
arca.live###Lfnq5YV6zrq3QrwD
arca.live###zYa8sDuyBUyBIPa0
```
